api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -239,4 +282,12 @@ public InputStream getDefaultConfig()
         return Thread.currentThread().getContextClassLoader().getResourceAsStream( "default-main.conf" );
     }
 
+    @Override
+    public Properties getSystemProperties()
+    {
+        Properties props = new Properties();
```

Comment:
Should we append it to default properties? i.e., Properties props = new Properties( System.getProperties());

Reply:
This method name is a little misleading. What it really does is collect system property *contributions*, and these are added to the main system properties by iterating over SystemPropertyProvider instances in DefaultIndyConfigFactory.